src/native/libs/System.Native/pal_io.c

```diff
@@ -1179,15 +1195,40 @@ int32_t SystemNative_CopyFile(intptr_t sourceFd, intptr_t destinationFd, int64_t
         copied = ret == 0;
     }
 #endif
+#ifdef __NR_copy_file_range
```

Comment:
Will it ever be `false` here?

Reply:
Yes, on Unix platforms other than Linux.

Reply:
Ah ok, I read as if this block is also linux-only.